node/full.go

```diff
@@ -264,6 +264,7 @@ func initBlockSyncService(mainKV ds.TxnDatastore, nodeConfig config.NodeConfig,
 }
 
 func initBlockManager(signingKey crypto.PrivKey, nodeConfig config.NodeConfig, genesis *cmtypes.GenesisDoc, store store.Store, mempool mempool.Mempool, proxyApp proxy.AppConns, dalc *da.DAClient, eventBus *cmtypes.EventBus, logger log.Logger, blockSyncService *block.BlockSyncService, seqMetrics *block.Metrics, execMetrics *state.Metrics) (*block.Manager, error) {
+	nodeConfig.BlockManagerConfig.LazyAggregator = nodeConfig.LazyAggregator
```

Comment:
would be good to make sure these are always in sync.

Reply:
looks like there is no need to define `LazyAggregator` in `NodeConfig`

we could move it into `BlockManagerConfig` directly

Reply:
moved it into `BlockManagerConfig`